app/views/posts/index.html.erb

```diff
@@ -6,19 +6,37 @@
 </head>
 <body>
   <h1>All Posts by <%= @user.name %></h1>
+
+  <h3>Create a Post</h3>
+  <%= form_with(model: [@user, @user.posts.build], url: user_posts_path(@user)) do |form| %>
+    <div class="field">
+      <%= form.label :title %>
+      <%= form.text_field :title %>
+    </div>
+    <div class="field">
+      <%= form.label :text %>
+      <%= form.text_area :text %>
+    </div>
+    <%= form.submit "Create Post" %>
+  <% end %>
```

Comment:
- [ ] Please relocate this form for creating a post to the `views/posts/new.html.erb` file.
- [ ] Also note that you would have to create a `new` method in your `PostsController` which will serve up the view to create a new post.